### PR TITLE
Adding Webhook Quickstart Template

### PR DESCRIPTION
**Pull request by cb-davidlai**
### What changed? Why?

* Added `npm run start-webhook` for demonstrating Webhook creation ([preview](https://github.com/cb-davidlai/coinbase-sdk-nodejs/blob/dl_wh/quickstart-template/README.md))
* Added `/webhook` for demonstrating simple Webhook App setup ([preview](https://github.com/cb-davidlai/coinbase-sdk-nodejs/blob/dl_wh/quickstart-template/webhook/README.md))

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>1</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>2</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

File: quickstart-template/webhook/app.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+app.use(express.json());
+
+app.post('/callback', (req, res) => {
+    const data = req.body;
+    console.log(JSON.stringify(data, null, 4));
+    const response = {
+        message: 'Data received',
+        received_data: data
+    };
+    res.json(response);
```
Comment (alex-stone) [inline quickstart-template/webhook/app.js:13]: Can we show examples of how they can use the webhook events?

File: quickstart-template/webhook/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+To setup a temporary public URL that points to this local webhook app,
+you can use [Pinggy](https://pinggy.io/) in another terminal window:
+```bash
+ssh -p 443 -R0:localhost:3000 -L4300:localhost:4300 qr@a.pinggy.io
+```
+You can also use [Vercel](https://vercel.com/) or other hosting solutions for your webhook app.
+
+Once the public webhook app is setup, copy the URL provided and test it with:
+```bash
+curl -X POST -H "Content-Type:application/json" -d '{"app": "webhook"}' http://url.a.free.pinggy.link:3000/callback
```
Comment (chaoyaji-cb) [inline quickstart-template/webhook/README.md:32]:
```suggestion
curl -X POST -H "Content-Type:application/json" -d '{"app": "webhook"}' {url_copied_from_pinggy_io}/callback
```

File: quickstart-template/webhook/app.js (hunk 1 above)
Comment (cb-davidlai) [reply quickstart-template/webhook/app.js:13]: we'll be showing more use cases later

**Pull request closed without merging**